src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -186,11 +199,18 @@ export default {
       type: String,
       default: 'No results found',
     },
+    isMultiple: {
```

Comment:
I think this is something we should discuss in the standup. Basically this component is not aware of that there is multiple options available, the whole logic is in ec-dropdown.vue. I understand that you need to know here what mode the parent is in but that's the problem. Child should not be aware of the parent. Three options how to fix this:
1. Move multiple logic into this component
2. Remove the support of multiple options completely, follow YAGNI principle. The multiple options is speced but not used anywhere in our code. We have to maintain it and I see that it complicates lots of logic in this ticket. I implemented it because it was 5 minute task while implementing ec-dropdown. We can remove that functionality and rework it when needed. 
3. We can scope out the multiple support from this ticket and do it separately with option 1.

^^ @matallo